tests/scripts/get_vault_data.cdc

```diff
@@ -0,0 +1,23 @@
+import ExampleToken from "ExampleToken"
+import FungibleTokenMetadataViews from "FungibleTokenMetadataViews"
+import MetadataViews from "MetadataViews"
+
+pub fun main(address: Address): Bool {
+    let account = getAccount(address)
+
+    let vaultRef = account
+        .getCapability(ExampleToken.VaultPublicPath)
+        .borrow<&{MetadataViews.Resolver}>()
+        ?? panic("Could not borrow a reference to the vault resolver")
+
+    let vaultData = FungibleTokenMetadataViews.getFTVaultData(vaultRef)
+        ?? panic("Token does not implement FTVaultData view")
+
+    let vault <- vaultData.createEmptyVault()
+
+    let vaultIsEmpty = vault.balance == 0.0
+
+    destroy vault
+
+    return vaultIsEmpty
```

Comment:
Not sure what you're doing here. You get the `vaultData` view, then don't use it and instead just return a boolean about whether a newly created empty vault is empty. Should this be different?

Reply:
I have reworked this test in https://github.com/onflow/flow-ft/pull/137/commits/d7a27dc816bf01cad2082ca4aeaec2b781e9d969#diff-3548e7dc61554c9b9f3f368ac783f2101b3bc8c61171c5ef974a51561b3ab3daR22-R35. Basically, I exercise the `vaultData.createEmptyVault()` closure, in order to increase the coverage, and make sure that the closure actually works as expected.